linux/keyman-config/keyman_config/uninstall_kmp.py

```diff
@@ -28,63 +31,55 @@ def delete_dir(dir: str) -> bool:
     return True
 
 
-def uninstall_kmp_shared(packageID):
-    """
-    Uninstall a kmp from /usr/local/share/keyman
+def _uninstall_dir(what, dir):
+    if os.path.isdir(dir):
+        if not os.access(dir, os.X_OK | os.W_OK):
+            error_dirs.add(dir)
+            return
 
-    Args:
-        packageID (str): Keyboard package ID
-    """
-    kbdir = get_keyboard_dir(InstallLocation.Shared, packageID)
-    if not os.path.isdir(kbdir):
-        msg = _("Keyboard directory for %s does not exist." % packageID)
-        logging.error(msg)
-        return msg
+        _delete_dir(dir)
```

Comment:
Would it be useful to check the status from _delete_dir?

The original code didn't use it though